labs_api/main.py

```diff
@@ -40,13 +40,13 @@ def _ping() -> str:
         logger.info("Pong!")
         return "pong"
 
-    @app.get("/market-insights/")
-    def _market_insights(market_id: HEX_ADDRESS_VALIDATOR) -> MarketInsightsResponse:
-        """Returns market insights for a given market on Omen."""
-        insights = market_insights(market_id)
-        logger.info(f"Insights for `{market_id}`: {insights.model_dump()}")
+    @app.get("/question-insights/")
```

Comment:
Don't you need the same thing for the `market-invalid` endpoint? But it is fine by me to implement it once you need it, if not yet.